docs/CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Ready to contribute? Here's how to set up `aicsimageio` for local development.
 2.  Clone your fork locally:
 
     ```bash
-    git clone git@github.com:{your_name_here}/aicsimageio.git
+    git clone https://{your_name_here}@github.com/aicsimageio.git
```

Comment:
Why can't git be used here?

Reply:
Personal Access Token authentication for git commands from the command line only accepts HTTPS (https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token), so that seems to be the recommended way now?

Reply:
The `git@` isn't a PAT it's SSH. SSH isn't going anywhere but I think the default on GitHub is to clone with https so I am personally fine with this change. GitHub default is fine to not confuse anyone imo.

Reply:
Huh, okay thanks!

Reply:
Sorry, I don't know whether I'm confused or I didn't explain it right.

Github is going to stop accepting passwords when using git commands from the command line (e.g. `git push`). Instead, one has to create PATs, and use those instead of the password. At the same time, SSH access is going to be deprected (https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token):

> Personal access tokens can only be used for HTTPS Git operations. If your repository uses an SSH remote URL, you will need to switch the remote from SSH to HTTPS.

Reply:
Wow. This is news to me. Why would SSH be deprecated?? Alright well good to know.